tests/integration/_experimental/object_detection/test_evaluator_multiclass_fixed.py

```diff
@@ -718,47 +718,50 @@
         y_label="F1-Score",
         curves=[
             Curve(
-                x=[0, 0.1, 0.4, 0.6, 0.7, 0.8, 0.9, 1],
-                y=[32 / 39, 15 / 19, 28 / 37, 13 / 18, 24 / 35, 12 / 17, 0.6, 0.4],
+                x=[0, 0.1, 0.4, 0.6, 0.8, 0.9, 1],
```

Comment:
Any idea how this test passed in previous PR?

Reply:
I have zero idea why they passed pre-merge, and correctly failed post-merge.